APTITUDE/APTITUDETEST.Data/Data/UserRepository.cs

```diff
@@ -327,7 +328,101 @@ public async Task<JsonResult> Update(UserVM user)
             }
         }
 
+        #endregion
 
+        #region
+        public async Task<JsonResult> RegisterUser(UserVM registerUser)
+        {
+            try
+            {
+                User? users = await Task.FromResult(_appDbContext.Users.Where(x => x.Email.Trim().ToLower() == registerUser.Email.Trim().ToLower() && x.IsDeleted != true).FirstOrDefault());
+                MasterCollege? masterCollege = await Task.FromResult(_appDbContext.MasterCollege.Where(x => x.Id == registerUser.CollegeId).FirstOrDefault());
```

Comment:
use this query below the if(users != null) block

Reply:
resolved